Update options for styling component inputs (#649)
alphagov/accessible-autocomplete: README.md

@import "accessible-autocomplete";
```

> ![NOTE]
>

> > If you add extra styles to adjust the `.autocomplete__input` elements, be aware that when using the `autoselect` option, the component overlays it above a second `.autocomplete__hint` input. That element displays the completions when what the user typed matches the start of a suggestion. It may need adjustment to account for the changes you may have made to `.autocomplete__input` so that the text of both align properly.

Reviewer: Can simplify this a bit perhaps?

```suggestion
> When styling the `.autocomplete__input` element, be aware that the `autoselect` option will render a second `.autocomplete__hint` input element for suggestion text. Both elements should be styled to ensure suggestions exactly align with the typed input text.
```

### Other options

#### `inputClasses` (default: `null`)

> Type: `string`

Reviewer: ```suggestion
Type: `string | null`
```

> Adds custom html classes to the `input` elements of the component:
> - the one the user types in
> - the one used to display the completion hint when `autoselect` is set to `true` (see `hintClasses` hereafter)

Reviewer: Think we should link between `hintClasses` and `inputClasses`?

```suggestion
Adds custom html classes to the generated `input` element.

If [`autoselect`](#autoselect) is set to `true`, the option [`hintClasses`](#hintClasses) can be configured separately or it will default to the `inputClasses` value.
```

#### `hintClasses` (default: `null`)

Type: `string`

> Adds custom html classes to the hint element that appears when what the user typed matches the start of a suggestion (and `autoselect` is set to `true`).
> 
> If `inputClasses` is set, it'll default to the value of `inputClasses`. You can set an empty string `''` to prevent the `inputClasses` being set on the hint as well.

Reviewer: Likewise, linking from `hintClasses` to `inputClasses`?

```suggestion
Adds custom html classes to the additional `input` element used for suggestion hints.

If [`autoselect`](#autoselect) is set to `true`, the option [`inputClasses`](#inputClasses) will be used as the default value unless `hintClasses` is set to an empty string `''`.
```

Type: `string`

Adds custom html classes to the generated `input` element.

If [`autoselect`](#autoselect) is set to `true`, the option [`hintClasses`](#hintClasses) can be configured separately or it will default to the `inputClasses` value.

#### `hintClasses` (default: `null`)

Type: `string`

> Adds custom html classes to the additional `input` element used for suggestion hints.

Author: Sounds clearer. The `hint` naming confused me a little at start, so expliciting what it represents will be helpful. I'll amend the docs quickly and I _think_ that'll be all the feedback taken into account 💪🏻
